assignment/assignment1/client/src/scenes/loginPage/Form.tsx

```diff
@@ -117,18 +117,21 @@ const Form = () => {
     // }
     // formData.append("picturePath", values.picture!.name);
 
-    const loggedInRespone = await fetch(`http://localhost:${PORT}/auth/login`, {
-      method: "POST",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify(values),
-    });
-    const loggedIn = await loggedInRespone.json();
+    const loggedInResponse = await fetch(
+      `http://localhost:${PORT}/auth/login`,
+      {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(values),
+      }
+    );
+    const loggedIn = await loggedInResponse.json();
 
     onSubmitProps.resetForm();
     if (loggedIn) {
       dispatch(
         setLogin({
-          user: loggedIn.userWithoutPassword.user,
+          user: loggedIn.userWithoutPassword,
```

Comment:
need to remember the user here is not a state/user object, but an sql query result